### PR TITLE
Added a note on how to add keys and keystore certificate when using PEM format

### PR DESCRIPTION
**Pull request by ShubhamRwt**
Signed-off-by: ShubhamRwt <srawat@redhat.com>

_Select the type of your PR_

- Documentation

### Description

This PR adds a note on how to add the key and keystore certificate directly when setting up the client access using listeners
### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Write tests
- [ ] Make sure all tests pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md
- [ ] Supply screenshots for visual changes, such as Grafana dashboards

### CODE REVIEW

**Review by ppatierno — APPROVED** [review]

**Review by scholzj — COMMENTED** [review]
If you really want to cover this, you should probably show some example or something. Or maybe Paul can add some better description. In particular, you should probably start from the beginning which is the cert / key file. Instead you start by saying it should be single-line. But right in the next sentence you again talk about `each line`. Because I'm not sure this is really something what will be clear to people who have really no clue. And those who have probably do not need this new addition.

**Comment by ShubhamRwt** [comment]
Thanks for the review @scholzj. I will get in touch in Paul to make things more clear.

**Review by PaulRMellor — COMMENTED** [review]
I've made a suggestion with examples.

**Comment by ShubhamRwt** [comment]
@PaulRMellor Is it good now?

**Review by scholzj — APPROVED** [review]

File: documentation/modules/deploying/proc-deploy-setup-external-clients.adoc [hunk 1]
```diff
@@ -261,9 +261,18 @@ props.put(SslConfigs.SSL_KEYSTORE_CERTIFICATE_CHAIN_CONFIG, "_<user.crt_file_con
...
 +
-You can add the values of the certificates directly to the configuration. 
-Or you can reference the certificates from a secure storage location.
-
+Add the keystore certificate and the private key directly to the configuration.
+Add as a single-line format.
+Between the `BEGIN CERTIFICATE` and `END CERTIFICATE` delimiters, start with a newline character (`\n`).
+End each line from the original certificate with `\n` too.
++
+.Example
```
Comment (PaulRMellor) [inline documentation/modules/deploying/proc-deploy-setup-external-clients.adoc:269]:
```suggestion
.Example keystore configuration
```

**Review by PaulRMellor — APPROVED** [review]
Suggestion for title to make it more explicit what we're showing

**Pull request merged**